src/components/User/User.tsx

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import './User.css';
 
-const User = () => {
+interface UserProps {
+  name:string,
+  preferred_lang:string
+}
+
+const User: React.FC<UserProps> = (props) => {
```

Comment:
Wondering if we should be consistent with de structuring? Not sure if it matters or not, but could be helpful for the sake of readability. 

If we wanted to change, I think it would be something like:
`( { name,  preferred_lang, id} )`

Because I think we will need a unique key for each user component but I could be wrong.